### ISSUE TITLE
Interaction between as.SingleCellExperiment and SCT counts?

### ISSUE THREAD

**Issue by mr-september** [satijalab/seurat#3883]
I have just converted a standard PBMC 5k dataset through `as.SingleCellExperiment`, with all preprocessing, clustering, etc... done in Seurat (and SCTransform).

Inspecting the `sce` object, both `counts` and `logcounts` appear to be identical, and I haven't been able to find the SCT-normalized counts.

Does anyone know if the SCT-normalized counts are converted as intended? And if so, where should I look for it? And if not, is there a easy way to manually write it from the original `Seurat` object to the converted `sce` object?

**Comment by torkencz** [comment]
Hi, 
Is there any chance you converted the RNA assay instead of the SCT? I would try to specify the SCT in the as.SingleCellExperiment command.

**Comment by mr-september** [comment]
I have explored this further, and found that the default assay appears to be SCT. 
I changed assay by `assay="SCT"` and `assay="RNA"`, and the `SCT` parameter gave me identical `sce object` as without specifying, and `RNA` gave me a different `sce object`. 

However, I noted that in both cases, after conversion, both `counts` and `logcounts` of the `sce object` are identical. Does not affect my use-case, but may be something worth considering for others.
